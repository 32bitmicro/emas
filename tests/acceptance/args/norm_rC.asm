lw r0, r0
lw r0, r1
lw r0, r2
lw r0, r3
lw r0, r4
lw r0, r5
lw r0, r6
lw r0, r7
