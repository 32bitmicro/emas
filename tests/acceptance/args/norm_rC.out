@ 0x0000 : 0x4000  /  010 000 0 000 000 000  /  16384
@ 0x0001 : 0x4001  /  010 000 0 000 000 001  /  16385
@ 0x0002 : 0x4002  /  010 000 0 000 000 010  /  16386
@ 0x0003 : 0x4003  /  010 000 0 000 000 011  /  16387
@ 0x0004 : 0x4004  /  010 000 0 000 000 100  /  16388
@ 0x0005 : 0x4005  /  010 000 0 000 000 101  /  16389
@ 0x0006 : 0x4006  /  010 000 0 000 000 110  /  16390
@ 0x0007 : 0x4007  /  010 000 0 000 000 111  /  16391
